loopFibonacci.c

```diff
@@ -0,0 +1,36 @@
+/*Lloyd Page*/
```

Comment:
You didn't properly handle when the user inputs 0 or 1! Always remember the base cases!